### DISCUSSION TITLE
Lua Error when hovering abilities

### DISCUSSION THREAD

**Discussion by Looble001** [Gennoken/GennUI#60] [Q&A] [answered]
I'm having this error everytime I hover my mouse on an ability, could you help me?

Message: ...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:522: attempt to call upvalue 'GetSpellInfo' (a nil value)
Time: Fri Aug 16 22:09:37 2024
Count: 34
Stack: ...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:522: attempt to call upvalue 'GetSpellInfo' (a nil value)
[string "@Interface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua"]:522: in function <...nterface/AddOns/ElvUI_GennUI/modules/tooltipicon.lua:507>
[string "=(tail call)"]: ?
[string "=[C]"]: in function `securecallfunction'
[string "@Interface/AddOns/Blizzard_SharedXMLGame/Tooltip/TooltipDataHandler.lua"]:61: in function <...lizzard_SharedXMLGame/Tooltip/TooltipDataHandler.lua:56>
[string "@Interface/AddOns/Blizzard_SharedXMLGame/Tooltip/TooltipDataHandler.lua"]:81: in function <...lizzard_SharedXMLGame/Tooltip/TooltipDataHandler.lua:77>
[string "@Interface/AddOns/Blizzard_SharedXMLGame/Tooltip/TooltipDataHandler.lua"]:117: in function <...lizzard_SharedXMLGame/Tooltip/TooltipDataHandler.lua:107>
[string "=[C]"]: in function `SetAttribute'

**Comment by Gennoken** [comment, marked as answer]
Fixed in 2.89, will wait 2-3 days before release in case more bugs are found.
